hack/ocm-minikube-ramen.sh

```diff
@@ -0,0 +1,360 @@
+	# fatal: destination path '${1}' already exists and is not an empty directory.
+	set -e
+	git --git-dir ${1}/.git fetch https://github.com/ShyamsundarR/${1} pause-on-plchange:pause-on-plchange
+	make -C ${1} build-images TRAVIS_BUILD=0 DOCKER_HOST=${DOCKER_HOST}
+}
+#ocm_operator_build multicloud-operators-deployable
+#ocm_operator_build multicloud-operators-subscription
+unset -f ocm_operator_build
+ramen_samples_branch_name=shyam_test_benamar_update_placement_to_avr
+exit_stack_push unset -v ramen_samples_branch_name
+ramen_samples_branch_build()
+{
+	set -- ocm-ramen-samples ${ramen_samples_branch_name}
+	set +e
+	git clone https://github.com/RamenDR/${1}
+	# fatal: destination path 'ocm-ramen-samples' already exists and is not an empty directory.
+	set -e
+	git --git-dir ${1}/.git --work-tree ${1} checkout main -b ${2}
+	git --git-dir ${1}/.git --work-tree ${1} pull --ff-only https://github.com/ShyamsundarR/${1} test
+	git --git-dir ${1}/.git --work-tree ${1} pull --rebase https://github.com/BenamarMk/${1} update_placement_to_avr
+	git --git-dir ${1}/.git --work-tree ${1} push git@github.com:hatfieldbrian/${1} ${2}:${2}
+	git --git-dir ${1}/.git --work-tree ${1} checkout -
+}
+exit_stack_push unset -f ramen_samples_branch_build
+ramen_samples_branch_checkout()
+{
+	set -- ocm-ramen-samples ${ramen_samples_branch_name}
+	set +e
+	git clone https://github.com/RamenDR/${1}
+	# fatal: destination path 'ocm-ramen-samples' already exists and is not an empty directory.
+	set -e
+	git --git-dir ${1}/.git --work-tree ${1} checkout main
+	exit_stack_push git --git-dir ${1}/.git --work-tree ${1} checkout -
+	git --git-dir ${1}/.git --work-tree ${1} fetch https://github.com/hatfieldbrian/${1} ${2}:${2}
+	exit_stack_pop
+	git --git-dir ${1}/.git --work-tree ${1} checkout ${2}
+	exit_stack_push git --git-dir ${1}/.git --work-tree ${1} checkout -
+}
+exit_stack_push unset -f ramen_samples_branch_checkout
+ramen_samples_branch_checkout_undo()
+{
+	exit_stack_pop
+}
+exit_stack_push unset -f ramen_samples_branch_checkout_undo
+application_sample_namespace_and_s3_deploy()
+{
+	ramen_samples_branch_checkout
+	kubectl --context ${1} apply -f ocm-ramen-samples/subscriptions/busybox/namespace.yaml
+	kubectl --context ${1} apply -f ocm-ramen-samples/subscriptions/busybox/s3secret.yaml
+	ramen_samples_branch_checkout_undo
+}
+exit_stack_push unset -f application_sample_namespace_and_s3_deploy
+application_sample_namespace_and_s3_undeploy()
+{
+	ramen_samples_branch_checkout
+	kubectl --context ${1} delete -f ocm-ramen-samples/subscriptions/busybox/s3secret.yaml
+	date
+	kubectl --context ${1} delete -f ocm-ramen-samples/subscriptions/busybox/namespace.yaml
+	date
+	ramen_samples_branch_checkout_undo
+}
+exit_stack_push unset -f application_sample_namespace_and_s3_undeploy
+application_sample_deploy()
+{
+	ramen_samples_branch_checkout
+	kubectl --context ${hub_cluster_name} apply -f ocm-ramen-samples/minio-deployment.yaml
```

Comment:
Lets source minio S3 deployment from ramen hack dir, rather than samples. Samples is possibly for other users as well, and I would prefer we kept minio out of samples repository and in here instead.

Reply:
File copied from `ocm-ramen-samples` to `ramen/hack` and used instead.  Pull request https://github.com/RamenDR/ocm-ramen-samples/pull/3 removes it from `ocm-ramen-samples`.